package.json

```diff
@@ -16,12 +16,12 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.11.0",
-    "@bcrs-shared-components/bread-crumb": "^1.0.2",
```

Comment:
I removed the caret because we don't want to automatically use subsequent minor/patch versions of this component. Instead, we want to keep using v1.0.2 until we explicitly update this.